README.md

```diff
@@ -92,6 +94,14 @@ return a promise
 set the authentication information directly without contacting the mojang servers.
 `playerName`, `uuid`, `accessToken` and `userProperties` need to be provided.
 
+#### WrapClient.setCustom(minecraftArguments,mainClass[,libraries]) 
```

Comment:
why not put it in the constructor ?
it's better to avoid that kind of mutability after construction

Reply:
Yeah, I gotta hand it to you that makes more sense.
I will change it.